<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketPay</class>
 <widget class="QWidget" name="TicketPay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>购买</string>
  </property>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ticketId</string>
   </property>
  </widget>
  <widget class="QLabel" name="spLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>startPlace</string>
   </property>
  </widget>
  <widget class="QLabel" name="apLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>arrivePlace</string>
   </property>
  </widget>
  <widget class="QLabel" name="stLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>startTime</string>
   </property>
  </widget>
  <widget class="QLabel" name="atLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>arriveTime</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buyBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>购买</string>
   </property>
  </widget>
  <widget class="QLabel" name="priceLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>price</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>乘机人</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="telLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>联系方式</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="identLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>身份证号码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
